src/reducers/view.js

```diff
@@ -32,21 +32,60 @@ const displayedFolder = (state = null, action) => {
   }
 }
 
+const fileCount = (state = null, action) => {
```

Comment:
To avoid runtime exception, we should declare `action` with the default value `{}`, giving:

```js
const fileCount = (state = null, action = {}) => {
```